internal/ethapi/api.go

```diff
@@ -1584,6 +1584,12 @@ func (s *PublicTransactionPoolAPI) GetTransactionReceipt(ctx context.Context, ha
 		"logsBloom":         receipt.Bloom,
 	}
 
+	// Quorum
+	if receipt.RevertReason != "" {
+		fields["revertReason"] = receipt.RevertReason
```

Comment:
Once `RevertReason` has been set to `[]byte` the hex encoding should happen here